src/Umbraco.Web.UI.Client/src/common/services/focuslock.service.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    "use strict";
+
+    function focusLockService() {
+        var elementToInert = document.querySelector('#mainwrapper');
```

Comment:
It's possible this returns null - for example when the Authorize Upgrade login page is shown (potentially other places). So then elementToInert is null and we get hidden exceptions which do not bubble up so it looks like everything is just stuck.

Reply:
Should this use $document so it can be unit tested? https://docs.angularjs.org/api/ng/service/$document

Reply:
@Shazwazza Could you have a look in #8966 to see if I have addressed your concerns correctly please? :-)

Reply:
Yep i think that looks fine (assume you have tested and works ? :)